tests/xharness/Microsoft.DotNet.XHarness.iOS.Shared/TestProject.cs

```diff
@@ -88,7 +89,18 @@ public async Task CreateCopyAsync (ITestTask test = null)
 				doc.SetNode ("DocumentationFile", "bin\\$(Configuration)\\nunitlite.xml");
 			}
 			doc.ResolveAllPaths (original_path);
-
+			if (doc.IsDotNetProject ()) {
```

Comment:
I'll have to move this to dotnet/xharness, created an issue not to forget: https://github.com/dotnet/xharness/issues/128